client/dashboard/domains/domain-overview/settings.tsx

```diff
@@ -1,17 +1,31 @@
-import { DomainSubtype, DomainTransferStatus, type Domain } from '@automattic/api-core';
+import {
+	DomainSubtype,
+	DomainTransferStatus,
+	type Domain,
+	DomainDiagnostics,
+} from '@automattic/api-core';
 import { __experimentalVStack as VStack } from '@wordpress/components';
 import { __ } from '@wordpress/i18n';
 import { SectionHeader } from '../../components/section-header';
 import { SummaryButtonList } from '../../components/summary-button-list';
 import DomainConnectionSetupSummary from '../domain-connection-setup/summary';
 import DomainContactDetailsSettingsSummary from '../domain-contact-details/summary';
+import DomainDiagnosticsSettingsSummary from '../domain-diagnostics/summary';
 import DnsSettingsSummary from '../domain-dns/summary';
 import DomainForwardingSettingsSummary from '../domain-forwarding/summary';
 import DomainGlueRecordsSettingsSummary from '../domain-glue-records/summary';
 import DomainSecuritySettingsSummary from '../domain-security/summary';
 import NameServersSettingsSummary from '../name-servers/summary';
 
-export default function DomainOverviewSettings( { domain }: { domain: Domain } ) {
+export default function DomainOverviewSettings( {
+	domain,
+	domainDiagnostics,
+	isLoadingDiagnostics,
+}: {
+	domain: Domain;
+	domainDiagnostics: DomainDiagnostics | undefined;
+	isLoadingDiagnostics: boolean;
```

Comment:
`isLoadingDiagnostics` is equivalent to `domainDiagnostics === undefined`; as we already have `domainDiagnostics: DomainDiagnostics | undefined;`; this props seems redundant.